src/containers/TransactionPage/TransactionPage.js

```diff
@@ -81,12 +84,35 @@ const onDisputeOrder = (
     });
 };
 
+// Submit host info and close the host infor modal
+const onSubmitHostInfo = (
+  currentTransactionId,
+  transitionName,
+  onTransition,
+  onUpdateHostInfo,
+  setHostInfoModalOpen,
+  setHostInfoSubmitted
+) => async values => {
+  onUpdateHostInfo(values);
+
+  try {
+    await onTransition(currentTransactionId, transitionName, {});
+    setHostInfoModalOpen(false);
+    return setHostInfoSubmitted(true);
+  } 
+  catch(e) {
+    console.error(e)
+  }
```

Comment:
why are we catching this error and only log it out?

Reply:
I think the reason why makeTransition function in the TransactionPage.duck.js catch the error and then throw it again because: the error is first caught and then is already handled by the transitionError and log.error. Then it re-throw the error again to ensures that the error propagates up to the call stack. which means if there is any code that call makeTransition, it can also catch and handle the error if it chooses to do so. In this situation, I didn't want to handle the error again so I only logged it out.